src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs

```diff
@@ -170,7 +180,10 @@ private void ValidateTokens(HttpContext httpContext, AntiforgeryTokenSet antifor
                 deserializedRequestToken,
                 out message))
             {
-                throw new AntiforgeryValidationException(message);
+                var ex = new AntiforgeryValidationException(message);
```

Comment:
Should probably spell out `exception` and leave `ex` for tests. But I see some `catch (Exception ex)` in other repos so obviously this isn't a hard line.